po/git-novice.ja.po

```diff
@@ -4675,16 +4675,16 @@ msgid ""
 "---"
 msgstr ""
 "---\n"
-"title: Ignoring Things\n"
+"title: ファイルを無視する\n"
 "teaching: 5\n"
 "exercises: 0\n"
 "questions:\n"
-"- \"How can I tell Git to ignore files I don't want to track?\"\n"
+"- \"Git で追跡したくないファイルを指定するにはどうすればいいの？\"\n"
```

Comment:
Suggestions for `questions`, `objectives`, and `keypoints` are to make style consistent with [R gapminder translation](https://swcarpentry-ja.github.io/r-novice-gapminder/ja/)
```suggestion
"- \"Git で追跡したくないファイルを指定するにはどうすればよいですか？\"\n"
```